website/content/terminal/crypto/dd/liquidations/_index.md

```diff
@@ -0,0 +1,16 @@
+```
+usage: liquidations
+            [--export {csv,json,xlsx}] [-h]
+```
+
+Display the liquidation data for a certain cryptocurrency. [Source: https://coinglass.github.io/API-Reference/]
```

Comment:
Is this documentation ok? I would definitely request a review here to make sure it makes sense.

Reply:
Yea it's fine, if you want to make it better you can add the image you uploaded there as a sneak peek

https://user-images.githubusercontent.com/1673206/186202157-31d1f6df-5f08-4849-98df-abb17c513fe1.png